sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AadAuthenticationProperties.java

```diff
@@ -576,74 +663,17 @@ private void validateAuthorizationClientProperties(String registrationId,
                 case RESOURCE_SERVER:
                 case RESOURCE_SERVER_WITH_OBO:
                     properties.setAuthorizationGrantType(AadAuthorizationGrantType.ON_BEHALF_OF);
-                    LOGGER.debug("The client '{}' sets the default value of AADAuthorizationGrantType to "
-                        + "'on_behalf_of'.", registrationId);
+                    LOGGER.debug("The client '{}' sets the default value of AADAuthorizationGrantType to 'on_behalf_of'.", registrationId);
```

Comment:
The `properties.setAuthorizationGrantType` shouldn't be in this method, it should be returned from this method.